WordPress/wordpress-playground
# [Data Liberation] Add XML API, Stream API, WXR URL Rewriter API

A part of https://github.com/WordPress/wordpress-playground/issues/1894. Follows up on https://github.com/WordPress/wordpress-playground/pull/1893.

This PR brings in a few more PHP APIs that were initially explored outside of Playground so that they can be incubated in Playground. See the linked descriptions for more details about each API:

* XML Processor from https://github.com/WordPress/wordpress-develop/pull/6713
* Stream chain from https://github.com/adamziel/wxr-normalize/pull/1
* A draft of a WXR URL Rewriter class capable of rewriting URLs in WXR files

## Testing instructions

* Confirm the PHPUnit tests pass in CI
* Confirm the test suite looks reasonabel
* That's it for now! It's all new code that's not actually used anywhere in Playground yet. I just want to merge it to keep iterating and improving.
